bears/vcs/git/GitCommitBear.py

```diff
@@ -291,14 +291,15 @@ def check_issue_reference(self, body,
             Whether to enforce presence of issue reference in the body of
             commit message.
         """
+
         if not body_close_issue:
             return
 
         host = self.get_host_from_remotes()
         if host not in self.SUPPORTED_HOST_KEYWORD_REGEX:
             return
 
-        if body_close_issue_on_last_line:
+        if body_close_issue_on_last_line and body:
```

Comment:
We have to do it differently. Now if the body is empty, but we require a reference in the message, this isn't reported any more.
We need a test case for this so this won't happen again

Reply:
I didn't exactly get what you mean by "...but we require a reference in the message, this isn't reported any more."
What isn't reported anymore?